Docs: Fix canvas G11 updates (#85024)
grafana/grafana: docs/sources/panels-visualizations/visualizations/canvas/index.md

### Adjust connectors

You can adjust connectors, adding angles to them, to fit the canvas you're working in. When you move connected elements, the connector resizes to fit the space. To adjust a connector, click it to display the midpoint controls and move those as needed. To make a connector a straight line again, move the midpoint back until the midpoint controls disappear.

If you move a connector so that it's almost a right angle or a straight line, the connector snaps into that angle or into a straight line.

> <!-- TODO: Fix gif / use updated demo from what's new -->

Author: @nmarrs We can repurpose the short demo that Drew did here, but we'll have to dump it at some point for something more evergreen. It's gonna be weird by about 11.2 to be talking about "new features". What do you think?

Reviewer: Sounds good to me :)

Author: Let's wait until it's been fancied up by the content team to replace this gif. In the meantime I think we can merge this PR?

> <!-- TODO: Fix gif / use updated demo from what's new -->

Author: ```suggestion
<!-- TODO: Use updated demo from what's new when uploaded to YouTube -->
```

{{< video-embed src="/media/docs/grafana/2024-01-05-Canvas-Pan-&-Zoom-Enablement-Video.mp4" max-width="750px" caption="Canvas pan and zoom enablement video" >}}

#### Infinite panning

You can enable infinite panning in a canvas when pan and zoom is enabled. This allows you to pan and zoom the canvas and uncover larger designs.

{{% admonition type="note" %}}

> Infinite panning is an experimental feature that may not work as expected in all scenarios. For example, elements that are not top / left constrained may experience unexpected movement when panning.

Author: ```suggestion
Infinite panning is an experimental feature that may not work as expected in all scenarios. For example, elements that are not top-left constrained may experience unexpected movement when panning.
```

{{% /admonition %}}

<!-- TODO: Add gif -->

> 

Reviewer: made some additional updates to canvas docs (cc @imatwawana)

Author: On line 127, do you mean top or left constrained or top-left constrained?

Reviewer: Vertical constraint = top
Horizontal constraint = left

Author: Okay, hyphenated instead of slash to make it clearer.
